testData/config/crds/fun_v1alpha1_toy.yaml

```diff
@@ -125,13 +125,50 @@ spec:
             s3Log:
               description: Used this for testing fieldNames with number.
               properties:
+                address:
+                  description: Must be an IPv4 or IPv6 address
+                  type: string
                 bucket:
                   minLength: 1
                   type: string
+                prefix:
+                  description: Must be a value in range that agrees with type of "Address"
+                    (i.e., 1-32 for IPv4, 1-128 for IPv6).
+                  format: int64
+                  type: integer
+              required:
+              - address
+              - prefix
               type: object
             template:
               description: This is a comment on an object field.
               type: object
+            testconfig:
+              items:
+                anyOf:
+                - type: string
+                - type: number
+                - type: object
+                - type: "null"
+                - type: array
+                - type: boolean
+                properties:
+                  address:
+                    description: Must be an IPv4 or IPv6 address
+                    type: string
+                  bucket:
+                    minLength: 1
+                    type: string
+                  prefix:
+                    description: Must be a value in range that agrees with type of
+                      "Address" (i.e., 1-32 for IPv4, 1-128 for IPv6).
+                    format: int64
+                    type: integer
+                required:
+                - address
+                - prefix
+                type: object
```

Comment:
The ```inter``` field is defined as an Interface in the types.go file.  Why is this object included here with ```address```, ```prefix```, and ```bucket``` fields?

Reply:
@alegacy The `type: object` is getting included here because of this code https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/internal/codegen/parse/crd.go#L436

Reply:
Yes, I see where```type:object``` is coming from, but I was questioning where the ```properties``` subsection is coming from.

Reply:
Ok, ignore this whole comment... for some reason I thought that ```S3Log``` and ```TestConfig" where of different types.